iamy.go

```diff
@@ -85,3 +94,37 @@ func init() {
 	}
 	defaultDir = filepath.Clean(dir)
 }
+
+func performVersionChecks() {
+	currentIAMyVersion, _ := semver.Make(Version)
+	log.Printf("current version is %s", currentIAMyVersion)
+
+	fileBytes, err := ioutil.ReadFile(versionFileName)
+	if err != nil {
+		if !os.IsNotExist(err) {
+			log.Printf("could not read version file %s, ignoring: %s", versionFileName, err)
```

Comment:
I think this would be better as a Debug statement. Most people won't care about it.